demisto_sdk/commands/validate/file_validator.py

```diff
@@ -961,7 +969,10 @@ def validate_pack(self):
         click.secho("================= Validating other pack files =================", fg="bright_cyan")
         for file in pack_files:
             click.echo(f'\nValidating {file}')
-            self.run_all_validations_on_file(file, file_type=find_type(file))
+            file_type = find_type(file)
+            if file_type:
+                file_type = file_type.value
+            self.run_all_validations_on_file(file, file_type=file_type)
```

Comment:
I can, but this might require a larger change and considering we would like to deprecate the file_validator soon, I'm not sure this change is necessary.